specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintAssignment.json

```diff
@@ -293,6 +311,13 @@
             "name": "AssignmentLockMode",
             "modelAsString": true
           }
+        },
+        "excludedPrincipals": {
```

Comment:
>excludedPrincipals [](start = 9, length = 18)

Discussed offline - this is being added for one API-only customer who will upgrade to the latest SDK version to use the feature.